### PR TITLE
Narrative overview of x-factor model and link to calibration notebook

### PR DESCRIPTION
**Pull request by alibow**
I was giving Syl an overview of what we did for the x-factor model and figured it should be written down in the documentation rather than just our brains!

### CODE REVIEW

**Review by zmbc — APPROVED** [review]
Good to write this down! I still think it's not entirely clear to me, but maybe once we get this documented I'll refer back to this instead of forgetting how it works every few months 😄

File: docs/source/models/risk_effects/x_factor/index.rst [hunk 1]
```diff
@@ -121,6 +121,16 @@ Additional literature on relapse rates
...
...ref:`CIFF malnutrition project <2019_concept_model_vivarium_ciff_sam>`. As we have risk effects as informed by the literature and described below, a primary remaining question is how to inform the population-level risk exposure of the x-factor model. For the prior implementation, we ran the simulation under several different population-level exposure values for the x-factor. We wrote a custom observer using the interactive context to observe the proportion of simulants who recovered from SAM/MAM who had a subsequent episode of SAM/MAM in the following year (`see this notebook for details on the code <https://github.com/ihmeuw/vivarium_research_ciff_sam/blob/main/model_validation/interactive_simulations/model_5/calibration_function_test_run.ipynb>`_). We then selected the population-level exposure that yielded results that were most similar to the relapse rate statistics summarized in the literature above. However, this process was additionally complicated/limited by the need for a burn-in period to reach a steady state across wasting categories (as x-factor exposure will vary by wasting state), lack of age specificity, vague/uncertain validation targets, and process inefficiencies.
```
[the remark this answers is not in the document]
Comment (SylLutze) [reply docs/source/models/risk_effects/x_factor/index.rst:128]: This is what happens. The notebook tries to match the relapse rate to literature values. Having more folks with the environmental risk factor means more people get SAM/MAM and more people will ultimately have a relapse. So we are matching to relapse rates, not an RR value or similar.

**Review by SylLutze — APPROVED** [review]

File: docs/source/models/risk_effects/x_factor/index.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (zmbc) [reply docs/source/models/risk_effects/x_factor/index.rst:128]: Right, but it varies the exposure while keeping the risk effects of the X-factor constant. We surely don't have a data source for those risk effects either, right? We must have calculated them somehow.

[the remark this answers is not in the document]
Comment (alibow) [reply docs/source/models/risk_effects/x_factor/index.rst:128]: Oh, yeah I just meant "we have SOME values that we are using for the risk effects, but we still have to choose what to use for the risk exposure." Does that clarify? I'll update the wording if so!

[the remark this answers is not in the document]
Comment (zmbc) [reply docs/source/models/risk_effects/x_factor/index.rst:128]: I think that would help, yes! It would also be nice to have some documentation of the reasoning/thinking behind how you translated literature values into X-factor risk effects, since that is a key part I always forget, but I don't want to hold up this PR!

[the remark this answers is not in the document]
Comment (alibow) [reply docs/source/models/risk_effects/x_factor/index.rst:128]: Oh, that I was hoping would be more clear in what we already have written here. We take the effect values directly from a study on differential risk of MAM/SAM incidence among those who were recently released from SAM treatment versus community controls. We don't need to transform/derive -- it's just a bit of a confused concept as we are using this measure of risk effect for a static exposure rather than a dynamic "recent treatment" exposure... probably it's a bit of both but we don't really know how to disentangle them! I'll take another look and see if I can make this more clear, but does this help in the meantime?

[the remark this answers is not in the document]
Comment (zmbc) [reply docs/source/models/risk_effects/x_factor/index.rst:128]: That makes sense but I'm not sure I understand the _why_ behind it. We use the differential risk among those who were recently released, making the assumption that being recently released is a perfect proxy for the latent X-factor. By the same logic, I could imagine us saying that the X-factor exposure is equal to the SAM prevalence, but we don't.

[the remark this answers is not in the document]
Comment (alibow) [reply docs/source/models/risk_effects/x_factor/index.rst:128]: Yeah, I think that we had tested that at some point, but if I recall correctly it didn't validate to the other validation targets well and I think we suspected that it was due to the prevalence of SAM not corresponding well to the proportion of children who get SAM in a given year (a value that will also depend on magnitude of X-factor risk effect...) -- that was for a previous version of the model where we had kids cycling through wasting states at higher rates than our more recent wasting exposure model improvements though.

Basically the message I am hoping to get across is: this model was never perfect in either concept or practice! It was somewhat exploratory and ended up having a small impact on overall results in the model it was used and it should be rethought if we want to include it in future applications.
